src/components/radios/_radios.scss

```diff
@@ -130,6 +130,13 @@
     }
   }
 
+  .govuk-radios__divider {
+    @include govuk-font($size: 19);
+    display: block;
+    margin-bottom: govuk-spacing(2);
+    padding: 0 0 0 ($govuk-radios-size / 4);
```

Comment:
This is _visually_ centring the text, but it's relying on the string being 'or' - if you use a different string then it doesn't work. What about making this element the same width as the radio buttons / checkboxes and then use `text-align: center`? That way it should work correctly for short strings, and automatically fall back to left-aligning if the string is long enough that centring it doesn't make sense.

Reply:
the aim is for the left alignment to be where the line is in the conditional reveal. i've update the calculation
<img width="397" alt="screen shot 2018-07-03 at 09 14 23" src="https://user-images.githubusercontent.com/3758555/42207380-b3c5f3ba-7ea1-11e8-965d-003f1c06ede0.png">
<img width="433" alt="screen shot 2018-07-03 at 09 14 18" src="https://user-images.githubusercontent.com/3758555/42207381-b3de6634-7ea1-11e8-92b2-71155c878818.png">

Reply:
we agreed to go with Ollie's suggestions